app/component/customize-search.scss

```diff
@@ -19,6 +19,10 @@
   }
 }
 
+.component-is-disabled {
```

Comment:
"disabled" usually means that component is visible but cannot be manipulated. Maybe think of a better term: ignored?